install/kubernetes/helm/istio/charts/security/templates/enable-mesh-permissive.yaml

```diff
@@ -0,0 +1,16 @@
+{{- define "security-permissive.yaml.tpl" }}
```

Comment:
Where is this used ? Permissive should be the default, no need to add it.

Reply:
I forgot to rebased. This file was deleted in master. Should be gone now.